tests/rt/boot_rt_kernel.pm

```diff
@@ -14,13 +14,12 @@ use base 'opensusebasetest';
 use strict;
 use warnings;
 use testapi;
-use utils;
 use bootloader_setup 'boot_grub_item';
 
 sub run() {
     my $self = shift;
     $self->boot_grub_item(2, 3);
-    $self->wait_boot(bootloader_time => 120);
+    $self->wait_boot(bootloader_time => 120, in_grub => 1);
```

Comment:
Maybe this solution is not so robust eventually.
http://eris.suse.cz/tests/12639#step/boot_rt_kernel/3

```perl
    assert_screen "grub2";

    for my $i (1 .. ($menu1 - 1)) {
        wait_screen_change { send_key 'down' };
    }
    save_screenshot;
    send_key 'ret';

    for my $i (1 .. ($menu2 - 1)) {
        wait_screen_change { send_key 'down' };
    }
    save_screenshot;
    send_key 'ret';
}
```

so the screen shot is done after entering sub menu, and then after selecting the kernel to boot.

Reply:
This looks better, but I guess we could simply use `send_key_until_needle_match` with proper timeout between key presses.

Reply:
Sorry I have missed your comment. Eventually, I have kept grub handling as it is and I am handling login explicitly in the module.